src/components/dashboard/dashboardComponent.tsx

```diff
@@ -11,6 +11,8 @@ import { useAppSelector } from '../../context/hooks';
 import { currentUserData } from '../../context/userReducer';
 import { Link } from 'react-router-dom';
 import SubmitButton from '../shared/form/submitButton';
+import CustomHeader from '../generic/CustomHeader';
+import React from 'react';
```

Comment:
I'm wondering why are you importing React everywhere and not using it :D

Reply:
We have it in almost every file, because it was necessary.
https://dev.to/titungdup/you-no-longer-need-to-import-react-from-react-3pbj - we need to consider whether we are removing or leaving these imports

Reply:
> We have it in almost every file, because it was necessary.

But we always used react  >= 17

Reply:
So why are these imports in files written by you?

Reply:
I do not remember